doozer/doozerlib/cli/release_gen_assembly.py

```diff
@@ -620,7 +677,7 @@ def _generate_assembly_definition(self) -> dict:
                     "assembly": {
                         'type': self.assembly_type.value,
                         'basis': {
-                            'brew_event': self.basis_event,
+                            f'{self.runtime.build_system}_event': self.basis_event,
```

Comment:
Hmm not a big fan of `konflux_event` since this is just the epoch we're calculating via our own DB records. We can in-theory do the same thing for brew builds (it maybe less accurate?). So maybe something more generic? `cutoff_time` , also since `event` here means something very specific (monotonically increasing non-overlapping int counter in brew)

Reply:
Not sure how `cutoff_time` would be appropriate. It's true that we store a Unix epoch, but I would expect a timestamp when reading `time`. In this case I just chose the easiest path to get rid of `brew` names in Konflux assemblies. I do agree that `event` might be misleading as well. `latest_build_epoch`? `cutoff_epoch_ms`? `cutoff_timestamp_ms`? Not a big name-finding guru :laughing:

Reply:
Hmm yeah. How about `cutoff_event`? `event` is vague enough